spec/flags_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+
+RSpec.describe ColorLS::Flags do
+  FIXTURES = 'spec/fixtures'.freeze
+
+  subject { capture_stdout { ColorLS::Flags.new(*args).process } }
+
+  def capture_stdout
+    old = $stdout
+    $stdout = fake = StringIO.new
+    yield
+    fake.string
+  ensure
+    $stdout = old
+  end
+
+  context 'with no flags' do
+    let(:args) { [FIXTURES] }
+
+    it { is_expected.to_not match(/((r|-).*(w|-).*(x|-).*){3}/) } # does not list file info
+    it { is_expected.to_not match(/\.hidden-file/) } # does not display hidden files
+    it { is_expected.to_not match(/Found \d+ contents/) } # does not show a report
+    it { is_expected.to match(/a-file.+symlinks.+z-file/) } # displays dirs & files alphabetically
```

Comment:
Reduce redundancy in this line too.

Reply:
It doesn't seem redundant to me. It's checking that files and dirs are mixed together & sorted alphabetically. In other words, it should _not_ match this order...

```
a-file
z-file
symlinks
```

...or this one:

```
symlinks
a-file
z-file
```

Reply:
Hmm, this makes sense. Why does it show a different order in Travis CI build?

Reply:
How about adding just this specific `it` within a pending context? Opening an issue to work on this in a different PR, would be better. Have this as pending and let's merge the rest of the specs.

Reply:
Alright, how shall I mark it as todo?

Reply:
NVM, I understand